tests/Doctrine/Tests/ORM/ConfigurationTest.php

```diff
@@ -179,6 +179,8 @@ public function testEnsureProductionSettings()
     {
         $this->setProductionSettings();
         $this->configuration->ensureProductionSettings();
+
+        $this->addToAssertionCount(1);
```

Comment:
@sensorario it's not a debug message, tests that don't assertions are a problem and we should solve them.

There's no way to ensure that no exception was raised so if we want to check for that the idea is to increment the assertion count after the method that was supposed to not throw exception and

> If the interpreter gets to the point where the assertion count is incremented it means that no exceptions have been thrown and our test is successful.